src/components/navbar/NavbarLinks.js

```diff
@@ -17,25 +17,42 @@ import { navbarLinksStyle } from "./navbarStyle";
 
 function NavbarLinks({ ...props }) {
   const { classes } = props;
-  const links = props.config.links;
+  const links = props.config.links
+/*
+  Set auth to true only when isLoaded = true & isEmpty = false
+  There is probaly a better way to approach this...
+  But my skills and time are in short supply :-D */
+  const auth = useSelector(state => state.firebase.auth.isLoaded && !state.firebase.auth.isEmpty);
+/*
+  Filter Navbar Links
+  auth = true filter links (see navConfig),
+  auth = false render all links from config.*/
+  const navLink = (links) => {
+    if(auth){
+      return links.filter(link => link.auth === auth)
```

Comment:
With the links from the navConfig schema (1) If the auth variable is true - indicsating the user is logged in, then => (2) filter out any links where link.auth is true (meaning the link should be rendered even when the user is signed in.